ouroboros-consensus/src/Ouroboros/Storage/VolatileDB/Impl.hs

```diff
@@ -506,33 +504,26 @@ mkInternalState hasFS err parser n files =
     wrapFsError (hasFsErr hasFS) err $
       go Index.empty Map.empty Map.empty Nothing [] files
   where
+    -- | This is the file with the maximum 'FileId' in the db.
     lastFile = safeMaximumOn fst files
     newFileInfo mp newIndex =
       ( filePath newIndex
       , newIndex
-      , newIndex + 1
       , Index.insert newIndex FileInfo.empty mp
       , 0 )
 
-    errorMsg = unwords
-        [ "VolatileDB invariant violation:"
-        , "The impossible happened!"
-        , "A file was found with less than "
-        , show n
-        , " blocks, but there are no files in db!"]
-
     truncateOnError Nothing _ _ = return ()
     truncateOnError (Just _) file offset =
-        -- The handle of the parser is closed at this point. We need
-        -- to reOpen the file in 'AppendMode' now (parser opens with
-        -- 'ReadMode').
-        --
-        -- Note that no file is open at this point, so we can safely
-        -- open with 'AppendMode' any file, without the fear of opening
-        -- multiple concurrent writers, which is not allowed.
-        --
-        withFile hasFS file (AppendMode AllowExisting) $ \hndl ->
-            hTruncate hasFS hndl (fromIntegral offset)
+      -- The handle of the parser is closed at this point. We need
+      -- to reοpen the file in 'AppendMode' now (parser opens with
```

Comment:
The `ο` in reοpen is a o micron, replace it with an ASCII o :slightly_smiling_face:

Reply:
:man_facepalming: :man_facepalming: